Formula/termtunnel.rb

```diff
@@ -0,0 +1,20 @@
+class Termtunnel < Formula
+  desc "Cross-platform terminal tunnel tool"
+  homepage "https://github.com/beordle/termtunnel"
+  url "https://github.com/beordle/termtunnel/archive/refs/tags/version-1.7.2.tar.gz"
+  sha256 "cd4d7b02bbae8a992f839b3d699923b0af4425ca5b2af6207e7f130fd18f1c8b"
+  license "MIT"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", "."
+    system "make"
+    bin.install "termtunnel"
+  end
+
+  test do
+    shell_output "#{bin}/termtunnel"
+    assert_equal "", shell_output("#{bin}/termtunnel sleep 1")
```

Comment:
I dont see the usage of this test?

Reply:
This is a shell pty proxy, similar to sshpass. so just make sure the exitcode is 0.

Reply:
the main reason is the program is TUI and cannot test the main features in the script. but the functionality of its pty proxy can be verified simply. if we verify it completely in a script, then we need to use the expect tool. I don't know if there is such an example.